keyboards/handwired/polykybd/split72/keymaps/default/keymap.c

```diff
@@ -1564,6 +1564,7 @@ void post_process_record_user(uint16_t keycode, keyrecord_t* record) {
         case KC_LANG:
             if (IS_LAYER_ON(_LL)) {
                 g_local.lang = (g_local.lang + 1) % NUM_LANG;
+                save_user_eeconf();
```

Comment:
Not sure if this code path is used, but the call made sense here as well.

Reply:
Yes, when pressing the lang button once again we switch to the next available language